rllib/policy/dynamic_tf_policy_v2.py

```diff
@@ -660,6 +660,7 @@ def _init_optimizers(self):
     def maybe_initialize_optimizer_and_loss(self):
         # We don't need to initialize loss calculation for MultiGPUTowerStack.
         if self._is_tower:
+            self.get_session().run(tf1.global_variables_initializer())
```

Comment:
Why do we need to call Session.run() during policy init? / Why did we not need to do this before?